src/Transport/Topology/MetaModel/DefaultNamespaceNameToConnectionStringMapper.cs

```diff
@@ -15,7 +15,7 @@ public DefaultNamespaceNameToConnectionStringMapper(ReadOnlySettings settings)
 
         public EntityAddress Map(EntityAddress value)
         {
-            if (!value.HasSuffix) return value;
+            if (!value.HasSuffix || value.HasConnectionString) return value;
```

Comment:
suffix can be either be a namespace name, or a true connectionstring (depends on what user supplies in sendoption.SetDestination()) if it already is a connectionstring it should not try to map it